docs/components/concepts/what-is-camunda-8.md

```diff
@@ -85,15 +85,15 @@ With [Tasklist](/components/tasklist/introduction-to-tasklist.md), process owner
 
 With [Console](/components/console/introduction-to-console.md), teams can create, configure, manage, and monitor clusters for all environments from development to production. Additionally, Console offers control over organizational settings such as user management, roles, and insights into usage metrics.
 
-## How does Camunda Platform 8 compare to other solutions?
+## How does Camunda 8 compare to other solutions?
 
 ### End-to-end orchestration
 
 Design, automate, and improve all components of the business process across different technologies, systems, infrastructures, people, and devices.
 
 ### Open architecture
 
-Fit into diverse and complex enterprise environments and technology stacks with Camunda’s open and scalable architecture. It provides a highly scalable platform based on open components that can be easily integrated with most common technical architectures and frameworks.
```

Comment:
Whoops, this is a platform we want!

Reply:
In a case like this, do we want to see "The Camunda platform's open and scalable..." or "Camunda Platform's open and scalable..."?

Reply:
Second sentence.

Lowercase "platform" -> It provides a highly scalable _**platform**_ based on open components that can be easily integrated with most common technical architectures and frameworks.

It's still a platform, just not a Platform (with the branding and product implications).

Reply:
Got it!

Reply:
Adjusted in latest commit 👍